### ISSUE TITLE
Project home and documentation

### ISSUE THREAD

**Issue by tbussmann** [perseas/Pyrseas#136]
As it seems, pgFoundry.org is gone. Here and on [pgxn](http://pgxn.org/dist/pyrseas/)  http://pyrseas.projects.pgfoundry.org/ still is mentioned as the home of the project. Is there a new location anywhere or will GitHub become the central place? Furthermore there is a reference to http://www.pyrseas.org/ here in the [Wiki](https://github.com/perseas/Pyrseas/wiki) which seems not to be related at all. 
I'd love to see https://pyrseas.wordpress.com/feature-matrix/ being updated and referenced or integrated in http://pyrseas.readthedocs.org/ so that we get an recent overview over PostgreSQL version / feature compatibility.
Please let me know if and how I can help with these tasks. Pyrseas is a project I really like but it's hard to follow if the links and docs are inconsistent or outdated.

**Comment by jmafc** [comment]
@tbussmann, I had not been made aware of the demise of pgfoundry.org. Last August, Marc Fournier had sent an email to PgFoundry users inquiring whether anyone was actively using it. I replied and he responded that he was "going to spend some time looking into how to go about rebuilding with miminal loss" and that he would let me know about it. I've just sent him an email to see what's transpired.

Assuming the worst, i.e., that pgfoundry.org is dead and won't be resurrected, yes, I believe GitHub should become the central place. Aside: pyrseas.org was a domain originally owned by myself, which simply pointed to the pgFoundry-hosted "website" (a couple of pages and links to GH and RTD), but I later dropped it since it wasn't worth the expense.

The pgxn reference comes from the `META.json` file and PyPI gets the same info from `setup.py` so those will have to be updated once I hear back from Marc and we decide what will be our new homepage. `docs/install.rst` will also have to be updated. And we'll have to decide what to do about a mailing list.

I have tried to keep the feature matrix page updated from time to time, but I see that the more recent PG/Python releases aren't mentioned. What gets shown in RTD comes from the `docs` directory so feel free to browse that and suggest a location and format where the feature matrix info could be incorporated.

I have been planning on doing a blog post on the future of Pyrseas itself, but it seems it's permanently stuck at the bottom of my priority list. If you're interested, perhaps we can discuss further, either privately or here.

**Comment by jmafc** [comment]
I just got a reply from Marc, saying he had moved pgFoundry to "a faster server / file system, [but] failed to update the IP apache was binding to".

In any case, although http://pyrseas.projects.pgfoundry.org/ is again accessible, I'm going to leave this issue open and work towards fixing the problems you raised and possibly moving away from pgFoundry. Your input will be appreciated.

**Comment by jmafc** [comment]
I've fixed the reference to pyrseas.org in the [Wiki](https://github.com/perseas/Pyrseas/wiki). Please let me know if there's any other such old reference that you've noticed.

**Comment by jmafc** [comment]
@tbussmann Are you still interested in helping? If so, there is one task that has just come up that may enable you to get your feet wet, so to speak: RTD has just decided to host projects under subdomains of readthedocs.io; so although http://pyrseas.readthedocs.org/en/latest/index.html redirects to https://pyrseas.readthedocs.io/en/latest/index.html, it would be helpful if we changed our links from the former to the latter.

**Comment by jmafc** [comment]
@dvarrazzo @rhunwicks @rdunklau @vayerx @andreypopp 
Gentlemen, this issue was opened over a year ago by @tbussmann. I'm reaching out to you because of your contributions to this project.

Back then I contacted Marc Fournier, who ran pgfoundry and it eventually became accessible. However, pgfoundry appears to be down again so I believe it's time to officially move the project. Unless someone has another suggestion, I think the most appropriate place would be GitHub Pages, i.e., something like http://perseas.github.io/. If anyone has experience with that or any recommentations on instructions for setup, your input will be most appreciated.

The one thing that pgfoundry had (and GitHub doesn't) was mailing list support. Granted the list never had much traffic but for some reason some people only interacted with the project via the list. If you have a recommendation for a replacement or alternative, either mail or online chatroom or similar please let us know.

**Comment by jmafc** [comment]
Initial http://perseas.github.io/ page created.

**Comment by dvarrazzo** [comment]
On Thu, Aug 24, 2017 at 1:17 AM, Joe Abbate <notifications@github.com>
wrote:

> @dvarrazzo <https://github.com/dvarrazzo> @rhunwicks
> <https://github.com/rhunwicks> @rdunklau <https://github.com/rdunklau>
> @vayerx <https://github.com/vayerx> @andreypopp
> <https://github.com/andreypopp>
> Gentlemen, this issue was opened over a year ago by @tbussmann
> <https://github.com/tbussmann>. I'm reaching out to you because of your
> contributions to this project.
>
> Back then I contacted Marc Fournier, who ran pgfoundry and it eventually
> became accessible. However, pgfoundry appears to be down again so I believe
> it's time to officially move the project. Unless someone has another
> suggestion, I think the most appropriate place would be GitHub Pages, i.e.,
> something like http://perseas.github.io/. If anyone has experience with
> that or any recommentations on instructions for setup, your input will be
> most appreciated.
>
> The one thing that pgfoundry had (and GitHub doesn't) was mailing list
> support. Granted the list never had much traffic but for some reason some
> people only interacted with the project via the list. If you have a
> recommendation for a replacement or alternative, either mail or online
> chatroom or similar please let us know.
>

PgFoundtry is a travesty. I have a couple of project locked up there (pgxn
client, pgmp), ssh access is broken and nobody ever bothered answering to
my requests for help.

github pages work alright, I use them for pg_repack (see
https://github.com/reorg/reorg.github.com if you find anything useful).
Pythonhosted is a goner too instead (just today I removed psycopg docs from
there). ReadTheDocs is supposed to be the next thing, but github is fine.

You can create a Google Group instead of a mailing list. ISTM the closest
thing. Expect people to open bugs or to write on stackoverflow instead of
writing to the list though. We live in dark times.

-- Daniele

**Comment by jmafc** [comment]
Thanks for the comments, Daniele. The last sentence made me LOL (particularly two days after the total eclipse--and other events here in the U.S.). The Pyrseas docs are already on https://pyrseas.readthedocs.io/en/latest/ so we'll link to those from the GH page. Part of the problem will be to style the different sites/pages somewhat consistently.

**Comment by jrschumacher** [comment]
@jmafc would you put that link in the project description ? Currently it points to pgfoundry. 

Also are there any references to example projects?

**Comment by jmafc** [comment]
@jrschumacher Done. Also fixed it in a couple of other files.

I'm afraid we haven't collected reference projects. If I'm not mistaken, @rhunwicks has used it in projects for his customers, @dvarrazzo has used it in some work projects, and somebody in Spain also tried it (and was the instigator of the ability to copy static files) but unfortunately then his employer chose to switch DBMS to MS SQL. Also I think when @rdunklau contributed to the project he was involved in the Multicorn FDW extensions but I don't in what capacity Pyrseas was used (or if it was used). The only other thing I could add is that our functional tests use the Pagila sample database which uses many Postgres features.

**Comment by rhunwicks** [comment]
@jmafc I no longer work for the organization where we used Pyrseas as our primary method of version controlling the database, so I can't provide you with any reference sites.

With regard to the move to Github, I think you should just deprecate the mailing list and encourage people to move discussion to specific GitHub issues. I think that (especially for projects with relatively low traffic) having too many communication channels spreads it all too thin - it is not clear to new users what is the appropriate channel because they all seem low traffic, and not all users check all channels. If the main source of traffic on the mailing list is discussing specific defects or feature requests, then I think Github Issues is a more functional platform.

**Comment by jmafc** [comment]
Thanks, @rhunwicks. That makes a lot of sense.

**Comment by jmafc** [comment]
Message to pyrseas-general@lists.pgfoundry.org was returned undeliverable. A similar message has now been sent to the list members directly informing them of the move of the home page.

@tbussmann I'm going to close this issue now. If there are specific items that you believe still need to be addressed, please open separate issues. Thanks.

**Comment by jmafc** [comment]
@tbussmann I've just submitted the first half of documentation changes in preparation for release 0.8.0. Thanks to readthedocs, it's already available at https://pyrseas.readthedocs.io/en/latest/. If you have some spare cycles, your review would be appreciated. I've linked to the WP Feature Matrix page, under "Known Issues", but I'm still not sure how we could incorporate them directly into the docs (WP and Sphinx are very different animals). If you have any suggestions on how to integrate them or to add to our barebones https://perseas.github.io/ site, please let me know.

**Comment by xenomachina** [comment]
Hello. I found out about this project while searching for a way to version my Postgresql schema. Looks good so far!

Are the docs at https://pyrseas.readthedocs.io/en/latest/ the best docs to read at this point? If so, it might be a good idea to link to them from the README and also from the https://perseas.github.io/ site.

When I didn't see a link in the README I tried going to the "docs/index.rst" directory in the code, which almost works, but Github does not render the `toctree` elements correctly. I was actually going to file an issue about this when I found this issue while searching to make sure I wasn't filing a duplicate.
